plugins/cliconf/os10.py

```diff
@@ -66,7 +66,7 @@ def get_device_info(self):
         return device_info
 
     @enable_mode
-    def get_config(self, source='running', format='text', flags=None):
+    def get_config(self, source='running', flags=None, format='text'):
```

Comment:
```suggestion
    def get_config(self, source='running', flags=None, format='text'):
```
if i hadn't changed the order, sanity tests would've failed complaining about that the flag parameter has been renamed to format and format has been renamed to flag...